packages/mediapipe-task-text/lib/src/interface/tasks/language_detection/language_detection.dart

```diff
@@ -0,0 +1,7 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
```

Comment:
Should this be a `library`?

Reply:
I don't think so. (My understanding is that) that's just an anchor for comments, but we don't have any here.